rustls/src/common_state.rs

```diff
@@ -274,6 +278,28 @@ impl CommonState {
         Ok(self.write_fragments(outgoing_tls, fragments))
     }
 
+    pub(crate) fn validate_rpk_negotiation(
```

Comment:
Would prefer to avoid the `rpk` acronym in favor of something more explicit, like `raw_key` (both in the name and in bindings inside it). Also it looks like this doesn't actually use `self`, make it a method on `RawKeyNegotationParams` instead?